packages/mgt-chat/src/utils/mentions.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { PersonCardInteraction } from '@microsoft/mgt-components';
+import { MgtTemplateProps, Person } from '@microsoft/mgt-react';
+import { ChatMessageMention, User } from '@microsoft/microsoft-graph-types';
+import { GraphChatClient } from 'src/statefulClient/StatefulGraphChatClient';
+import { Mention } from '@azure/communication-react';
+
+export const renderMGTMention = (chatState: GraphChatClient) => {
+  return (mention: Mention, defaultRenderer: (mention: Mention) => JSX.Element): JSX.Element => {
+    let render: JSX.Element = defaultRenderer(mention);
+
+    const mentions = chatState?.mentions ?? [];
+    // TODO: Array.flat() is a new api, update?
```

Comment:
Array,prototype.flat() is part of es2019 and has full support in all modern browsers, I think we we're good to use it here.